package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "@lukeed/ms": "^2.0.1",
     "fastify-plugin": "^4.0.0",
-    "tiny-lru": "^11.0.0"
```

Comment:
This is a breaking change.

Reply:
Why would that be? I have the same explanation as below, the LocalStore is our implementation that cannot be overridden – it behaves the same